zc_plugins/POSM/v6.0.0/admin/includes/classes/observers/auto.PosmHelp.php

```diff
@@ -0,0 +1,16 @@
+<?php
+class zcObserverPosmHelp extends base {
+  function __construct() {
+    $this->attach($this, array('NOTIFIER_PLUGIN_HELP_PAGE_URL_LOOKUP'));
+  }
+
+  function update(&$class, $eventID, $page, &$help_page) {
+     if ($eventID == 'NOTIFIER_PLUGIN_HELP_PAGE_URL_LOOKUP') {
+        if ($page == FILENAME_PRODUCTS_OPTIONS_STOCK) {
+           $help_page = 'https://docs.zen-cart.com/user/running/posm/'; 
+        } elseif ($page == FILENAME_PRODUCTS_OPTIONS_STOCK_VIEW_ALL) {
```

Comment:
It's the same help-page for both; use in_array to see if `$page` is either and set the help-page if true.